ssca-manager/contracts/openapi/v1/openapi.yaml

```
@@ -40,6 +40,105 @@ paths:
         $ref: '#/components/requestBodies/SbomProcessRequestBody'
       tags:
         - sbomProcessor
+  '/v1/org/{org}/project/{project}/normalized-sbom-component':
+    parameters:
+      - name: org
+        in: path
+        required: true
+        schema:
+          type: string
+        description: Harness Organization Id
+      - in: path
+        required: true
+        schema:
+          type: string
+        description: Harness Project Id
+        name: project
+    get:
+      summary: Get Normalised SBOM Components
```

Use sentence case globally. Some descriptions are also starting with lowercase.

For example (but apply everywwhere):
```suggestion
      summary: Get normalised SBOM components
```